modules/installation-azure-stack-hub-config-yaml.adoc

```diff
@@ -12,6 +24,7 @@ You can customize the `install-config.yaml` file to specify more details about y
 This sample YAML file is provided for reference only. Use it as a resource to enter parameter values into the installation configuration file that you created manually.
 ====
 
+ifdef::ash[]
```

Comment:
@codyhoag In the published output, I am getting content for both ash and ash-custom. I am sure it is something mundane that I am looking past. Appreciate you looking at this.